CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- `showClearAllFilters` prop to the `FilterNavigator`.
+- `showSelectedFiltersInTitle` prop to the `FilterNavigator`.
```

Comment:
Hey @klzns @victorhmp , would be good if only exists one line?
```suggestion
- `showClearAllFilters`, `showSelectedFiltersInTitle` props to the `FilterNavigator`.
```

Reply:
Could be! No problem :)